checklist.html

```diff
@@ -396,44 +459,59 @@ <h2 id="label-lists">Lists</h2>
   </p>
-  <dl class="checklist">
-
+  <div class="checklist">
     <details id="section-controls-01">
       <summary>
         Use the <code>a</code> element for links.
       </summary>
       <div class="checklist-details">
-        <p class="checklist-criterion"><a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html">1.3.1 Info and Relationships</a></p>
-        <p class="checklist-description">Links are used for loading a new page, or shifting keyboard focus from one element to another.</p>
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html">1.3.1 Info and Relationships</a>
+        </p>
+        <p class="checklist-description">
+          Links are used to load a new page, or shift keyboard focus from one element to another.
+        </p>
       </div>
     </details>
     <details id="section-controls-02">
       <summary>
         Use the <code>button</code> element for buttons.
       </summary>
       <div class="checklist-details">
-        <p class="checklist-criterion"><a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html">1.3.1 Info and Relationships</a></p>
-        <p class="checklist-description">Buttons are used to submit data or perform an on-screen action which does not shift keyboard focus.</p>
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html">1.3.1 Info and Relationships</a>
+        </p>
+        <p class="checklist-description">
+          Buttons are used to submit data or perform an on-screen action which does not shift keyboard focus.
+        </p>
       </div>
     </details>
     <details id="section-controls-03">
       <summary>
-        Provide a skip link and ensure that it is visible when focused.
+        Provide a skip link and make sure that it is visible when focused.
       </summary>
       <div class="checklist-details">
-        <p class="checklist-criterion"><a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html">2.4.1 Bypass Blocks</a></p>
-        <p class="checklist-description"><a href="/posts/skip-nav-links/">Skip links</a> provide quick access the main page content, allowing a person to quickly and easily bypass repeated content such as a site's main navigation.</p>
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html">2.4.1 Bypass Blocks</a>
+        </p>
+        <p class="checklist-description">
+          <a href="/posts/skip-nav-links/">Skip links</a> provide quick access to the main page content. This allows a person to quickly and easily bypass repeated content such as a site's main navigation.
```

Comment:
A skip link can be used to provide quick access to the main content of a page or view. This will allow a person to easily bypass globally repeated content like a website's primary navigation, or persistent search widget.